README.md

```diff
@@ -129,25 +128,61 @@ testsys run aws-k8s \
   --ec2-provider-image "ec2-resource-agent:eks"
 ```
 
-## Development
+### Test Results
 
-### Project Structure
+Executing the `testsys run` command will kick off the setup and execution of
+the Sonobuoy tests on an EKS cluster using Bottlerocket as the host OS for
+the cluster nodes.
 
-- `model` is the root dependency.
-It includes the CRDs and clients for interacting with them.
+To get the status and results of test execution, run the `testsys status`
+command for a high level summary:
 
-- `controller` contains the Kubernetes controller responsible for running resource and test pods.
+```shell
+ NAME           TYPE   STATE    PASSED   SKIPPED   FAILED
+ testsys-demo   Test   passed   1        5772      0
+```
 
-- `agent` contains libraries with the traits and harnesses for creating test and resource agents.
+When the test run has completed you may get the full logs of the test
+results. The content of the resulting tar file will vary depending on
+the tests being run.
+
+```shell
+testsys results --destination testresults.tar --test-name testsys-demo
+tar -xvf testresults.tar
+```
+
+### Cleanup
+
+If you used the `--cluster-destruction-policy never` as given above, there
+will be an EKS cluster running at the end of execution. This can be very
+convenient to keep around for subsequent test runs.
 
-- `bottlerocket-agents` contains the implementations of the test and resource traits that we use for Bottlerocket testing.
+It does however consume resources and incur usage charges. If you are
+done running tests and would like to clean up, run the following to
+delete the EKS cluster.
 
-- `testsys` contains the command line interface for installing the system and running tests.
+```shell
+eksctl delete cluster external-cluster
+```
+
+**Note:** This will destroy the EKS cluster, so ensure you are no longer
+using it and you have the correct cluster name before running the
+example command.
+
+## Development
+
+### Project Structure
+
+- `model` is the root dependency. It includes the CRDs and clients for interacting with them.
+- `controller` contains the Kubernetes controller responsible for running resource and test pods.
+- `agent` contains libraries with the traits and harnesses for creating test and resource agents.
+- `bottlerocket/agents` contains the implementations of the test and resource traits that we use for Bottlerocket testing.
+- `bottlerocket/testsys` contains the command line interface for installing the system and running tests.
 
 The `model`, `agents` and `controller` crates are general-purpose, and define the TestSys system.
-It is possible to use these libraries and controller to for testing purposes other than Bottlerocket.
+It is possible to use these libraries and controller for testing purposes other than Bottlerocket.
```

Comment:
Adding as a reminder for myself: `TestManager` provides and interface for managing a testsys cluster.